### DISCUSSION TITLE
Faikin synchronize with Onecta?

### DISCUSSION THREAD

**Discussion by Ulf349** [revk/xESP32-Faikout#415] [General]
Hello all, 
Thanks for this project! 
I have 10 Perfera devices. It works, but not so good with my iobroker over the Daikin cloud. This is the idea to implement the additional Faikin module to each Daikin.
If I change any value over Onecta, will it synchronize with Faikin mqtt and also in the opposite direction? 
Or is it same as I will use the IR control? 

Thanks for your support. 
BR Ulf

**Comment by revk** [comment]
Any setting we can see from the Daikin will be reported however it comes in.

**Reply by Ulf349** [reply]
Sorry revk for double question, the Faikin and Daikin Cloud are permanent synchronized?

**Reply by revk** [reply]
No Faikin and the actual air-con are
